devops/cloudbuild-pr.yaml

```diff
@@ -35,7 +35,7 @@ steps:
   # Deploy to firebase channel, using the PR #
   # store log to /workspace to get the channel URL
   #
-  - name: gcr.io/yfthig-dev/firebase
+  - name: us-docker.pkg.dev/firebase-cli/us/firebase
```

Comment:
Megan said she'd update this for the other UIs. (BRD, Dashboard, Filings, Edit, Namerequest, BAR? Auth Web?)